scout/server/blueprints/variants/views.py

```diff
@@ -343,12 +348,14 @@ def cancer_variants(institute_id, case_name):
 
     variant_type = request.args.get("variant_type", "clinical")
     variants_query = store.variants(case_obj["_id"], category="cancer", query=form.data)
+    query = store.get_query(case_obj["_id"], form.data, None, category)
```

Comment:
Why don't you use the store.variants_count that you created in scout/adapter/mongo/variant.py to do that instead?